lib/web_ui/lib/src/engine/scene_view.dart

```diff
@@ -163,11 +163,13 @@ final class PictureSliceContainer extends SliceContainer {
       final DomCSSStyleDeclaration style = canvas.style;
       final double logicalWidth = roundedOutBounds.width / window.devicePixelRatio;
```

Comment:
Do we lose the sub-pixel delta between the rounded values and the actual values? I wonder of this is why we get https://github.com/flutter/flutter/issues/122541 in CanvasKit, and maybe we have the same issue in Skwasm?

Reply:
When I first implemented the platform views change (when it was a draft), I actually did have a ton of golden diffs because I had blurring due to what you're talking about. The rounding out of the rectangle fixed it, and got rid of the extra blurring and the goldens were consistent with previous behavior.

That being said, I am not 100% sure the existing behavior is totally correct. This change at least gets us back to the previous rendering behavior though. I think I will follow up and look a little deeper into this, just to make sure that we aren't ending up with any fractional weirdness here.